Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM gliderlabs/alpine:3.3
-ENTRYPOINT ["/bin/signalfx-janitor"]
+FROM debian
 
-RUN apk-install ca-certificates
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y update && apt-get install -y ca-certificates
+COPY build/batchcli-v0.0.12-linux-amd64/batchcli /usr/local/bin/batchcli
```

Comment:
ahem this should be https://github.com/Clever/cil-reliability-dashboard/blob/master/Dockerfile#L4